SPECS/libcap-ng/libcap-ng.spec

```diff
@@ -59,14 +74,16 @@ find %{buildroot} -type f -name "*.la" -delete -print
 
 %files devel
 %defattr(-, root, root)
-%{_libdir}/*.so
+%{_libdir}/*.so*
 %{_libdir}/pkgconfig/*.pc
 %{_includedir}/*.h
 %{_mandir}/man3/*
 %{_datadir}/aclocal/*.m4
-%{_libdir}/*.a
```

Comment:
I'm in general in favour of removing static libraries but please double-check that all packages depending on `libcap-ng-devel` still build. I see you already ran a full build, so that should be good enough.:)

Reply:
For this one it was giving me
   File not found: /usr/src/azl/BUILDROOT/libcap-ng-0.8.4-1.azl3.x86_64/usr/lib/*.a"

Reply:
I changed stuff to get it back, thank you :)